extensions/web/06_streams.js

```diff
@@ -3475,7 +3635,11 @@
      * @param {R} chunk
      * @returns {void}
      */
-    enqueue(chunk) {
+    enqueue(chunk = undefined) {
```

Comment:
That is odd... is there a reason?

Reply:
Yup - `prototype.enqueue.length` will be wrong otherwise (1 instead of 0). Optional fields shouldn't count towards length, and this marks a field as optional in JS.